### ISSUE TITLE
New signup probably needs more testing for edge cases

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/hdx-ckan#3580]
In testing other issues, I've noticed a couple of things that seem odd.  

For example, deleted users can't be invited to orgs, an email that has never been used on HDX can be invited.  I've also seen previously deleted emails not being available for sign up.  There wasn't time to fully test all the possible permutations, but we probably still have some rough edges there we could work on when we have time.

**Comment by danmihaila** [comment]
I am not sure if the mentioned cases have something to do with the new onboarding. lets discuss on slack or skype

**Comment by amcguire62** [comment]
Can we identify what triggers the edge case. In parallel either Godfrey or myself is going to approach some of the these failed reg users to better understand why they could not complete their registration process.

**Comment by teodorescuserban** [comment]
Maybe this one will be split as a new issue: the signup process seem to use case sensitive email addresses, which makes possible the creation of multiple accounts with the same email.

For example:

teodorescu.serban@example.com
Teodorescu.Serban@gmail.com

**Comment by amcguire62** [comment]
Yes and i found another example of a failed registration which had a duplicate so for some its reasonable to assume this is causing an issue ...

**Comment by teodorescuserban** [comment]
Please note this is not a case-sensitive issue. I found duplicates having exactly the same letters (same capitalization).
